cluster-provision/gocli/cmd/run.go

```diff
@@ -574,19 +575,37 @@ func run(cmd *cobra.Command, args []string) (retErr error) {
 	// the VM console from the container without ssh
 	qemuArgs += " -serial pty"
 
+	arch := runtime.GOARCH
+	var qemuDevice string
+
+	// Use virtio-net-ccw device incase of s390x Architecture.
+	if arch == "s390x" {
+		qemuDevice = "virtio-net-ccw"
```

Comment:
Could we introduce constants that tell the reader what this is about?

i.e.

```suggestion
const (
    QEMU_DEVICE_S390X  = "virtio-net-ccw"
    QEMU_DEVICE_X86_64 = "virtio-net-pci"
)
```

and use these? Or create a func similar to `GetSSHUserForArchitecture`?

Reply:
Done